data/webshare-permission-policy.yml

```diff
@@ -13,16 +13,45 @@ symptoms:
 solutions:
   notes:
     - While the standard is not finished yet, adding the `allow="web-share"` attrubute to the iFrame makes it work in all browsers.
-  interventions:
-    - https://github.com/mozilla-extensions/webcompat-addon/blob/8b3865cb772b2596549705d11ed3aa1a376c60cd/src/data/injections.js#L572-L581
 
 references:
   breakage:
-    - https://github.com/webcompat/web-bugs/issues/68473
-    - https://github.com/webcompat/web-bugs/issues/88374
-    - https://github.com/webcompat/web-bugs/issues/105859
```

Comment:
Looks like https://github.com/webcompat/web-bugs/issues/105859 is a Safari only issue, so I didn't add it to this list, wdyt @denschub?

Reply:
Yeah, sounds good!